### PR TITLE
Code Review for FCC Test Suite

### PR DESCRIPTION
**Pull request by Whoistolu**
## Code Review Skills on the FCC- Test-Suite project from freecodecamp

In this Project. we:
- Added linters to test for code standards

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+                ></textarea>
+              </div>
+          
+              <div class="form-group">
+                <button type="submit" id="submit" class="submit-button">
+                  Submit
+                </button>
+              </div>
+            </form>
+          </div>
```
Comment (Whoistolu) [inline index.html:217]: I think it would be better to use semantic tags so as to clearly define the element's purpose.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+
+    <!--
+    Hello Camper!
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+    - The freeCodeCamp Team
+  
+    -->
```
Comment (Whoistolu) [inline index.html:231]: I think it would be better to remove the comments in order to have a clean code.

File: style.css [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+body {
+  font-family: 'Poppins', sans-serif;
+  font-size: 1rem;
+  font-weight: 400;
+  line-height: 1.4;
+  color: var(--color-white);
+  margin: 0;
+}
+
+/* mobile friendly alternative to using  background-attachment: fixed */
```
Comment (Whoistolu) [inline style.css:25]: I think it would be better to remove this comment also in order to have a clean code.

File: style.css [hunk 4]
```diff
@@ -0,0 +1,181 @@
...
+
+.form-control {
+  display: block;
+  width: 100%;
+  height: 2.375rem;
+  padding: 0.375rem 0.75rem;
+  color: #495057;
+  background-color: #fff;
+
+  /* background-clip: padding-box; */
```
Comment (Whoistolu) [inline style.css:147]: I think it would be better to remove this comment also in order to have a clean code.

**Review by Whoistolu — COMMENTED** [review]
## You have done a great job!

- You have a professional readme
- A professional Pull Request
But;
- Kindly go through the comments and fix them

**Review by mimipeshy — APPROVED** [review]
## Hi @Whoistolu , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**